src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/CommandHandlers/UpdateChangeOfPartyRequestCommandHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NServiceBus;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Data.Extensions;
+using SFA.DAS.CommitmentsV2.Messages.Commands;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.CommitmentsV2.MessageHandlers.CommandHandlers
+{
+    public class UpdateChangeOfPartyRequestCommandHandler : IHandleMessages<UpdateChangeOfPartyRequestCommand>
```

Comment:
Instead of Command can we use Event, i.e. CohortWithChangeOfProviderUpdated, but happy to discuss this further